internal/cmd/egctl/translate.go

```diff
@@ -55,6 +60,8 @@ func NewTranslateCommand() *cobra.Command {
 	}
 	translateCommand.PersistentFlags().StringVarP(&inType, "from", "", gatewayAPIType, getValidInputTypesStr())
 	translateCommand.PersistentFlags().StringVarP(&outType, "to", "", xdsType, getValidOutputTypesStr())
+	translateCommand.PersistentFlags().StringVarP(&output, "output", "o", yamlOutput, "One of 'yaml' or 'json'")
+	translateCommand.PersistentFlags().StringVarP(&resourceType, "type", "t", "all", "One of 'bootstrap', 'cluster', 'listener', 'route', 'all'")
```

Comment:
nit, thoughts on using existing style of `getValidResourceTypesStr()` ? allows you to use magic vars for `bootstrap` and define the string once